### PR TITLE
Finally caught up with your blog posts!

### PR DESCRIPTION
**Pull request by rderollepot**
... and found some typos along the way. But Pycharm spell checker found a lot more, credit to it!

### CODE REVIEW

File: _posts/2022-09-05-workflow-2.md [hunk 1]
```diff
@@ -78,25 +78,25 @@ I could have set him up with our whole development environment, as our team did
...
 
-Since we're already [packaging](/workflow-2#simulator) a nice executable for the simulator, the easiest thing is to reuse that one. So by simply giving read access on the simulator's server to the researcher, they can now grab any package and run it on their computer just by double-clicking the executable. And since the packaging script uses a [Discord webhook](https://discord.com/developers/docs/resources/webhook) to notify on completion, I don't even have to tell my colleague when a new package is available, it's all automated!
+Since we're already [packaged](/workflow-2#simulator) a nice executable for the simulator, the easiest thing is to reuse that one. So by simply giving read access on the simulator's server to the researcher, they can now grab any package and run it on their computer just by double-clicking the executable. And since the packaging script uses a [Discord webhook](https://discord.com/developers/docs/resources/webhook) to notify on completion, I don't even have to tell my colleague when a new package is available, it's all automated!
```
Comment (brifsttar) [inline _posts/2022-09-05-workflow-2.md]: Hmm, no? I think mine was right here. Not sure how to just revert that from GitHub's UI...

File: _posts/2023-03-01-mirrors.md [hunk 2]
```diff
@@ -70,15 +70,15 @@ With the recent progress on [realtime ray tracing](https://en.wikipedia.org/wiki
...
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve completed the <a href="https://twitter.com/hashtag/nDisplay?src=hash&amp;ref_src=twsrc%5Etfw">#nDisplay</a> configuration of our largest <a href="https://twitter.com/hashtag/DrivingSimulation?src=hash&amp;ref_src=twsrc%5Etfw">#DrivingSimulation</a> platform at <a href="https://twitter.com/LescotUniEiffel?ref_src=twsrc%5Etfw">@LescotUniEiffel</a>. 12 displays (including tricky mirrors) running on 8 computers. Easy to set up and free-of-charge thanks to <a href="https://twitter.com/hashtag/UE4?src=hash&amp;ref_src=twsrc%5Etfw">#UE4</a>!<br><br>Let&#39;s do some <a href="https://twitter.com/hashtag/mobility?src=hash&amp;ref_src=twsrc%5Etfw">#mobility</a> <a href="https://twitter.com/hashtag/research?src=hash&amp;ref_src=twsrc%5Etfw">#research</a> now! <a href="https://t.co/FnsmzSlqJT">pic.twitter.com/FnsmzSlqJT</a></p>&mdash; Bertrand Richard (@brifsttar) <a href="https://twitter.com/brifsttar/status/1458740786481307658?ref_src=twsrc%5Etfw">November 11, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (brifsttar) [inline _posts/2023-03-01-mirrors.md]: What changed here?

File: _posts/2023-05-01-whats-new-2023-05.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ We've got bigger plans in the long run for this simulator, and we're actively wo
...
 
-Since SIMAX is undergoing maintenance, it was also an opportunity to start testing this experiment with its real "production" hardware. It was running fine on a single desktop computer with a single viewport; but how about on an [nDisplay](/ndisplay/) cluster with 8 computers, 12 viewports, covering almost 360° of horizontal field of view plus [pesky mirrors](/mirrors/)? Well, everything runs fine! No major hitches or performance drop, we pretty much have the same performance on the cluster as we do on desktop.
+Since SIMAX is undergoing maintenance, it was also an opportunity to start testing this experiment with its real "production" hardware. It was running fine on a single desktop computer with a single viewport; but how about an [nDisplay](/ndisplay/) cluster with 8 computers, 12 viewports, covering almost 360° of horizontal field of view plus [pesky mirrors](/mirrors/)? Well, everything runs fine! No major hitches or performance drop, we pretty much have the same performance on the cluster as we do on desktop.
```
Comment (brifsttar) [inline _posts/2023-05-01-whats-new-2023-05.md]: Hmm not sure, mine seems valid here, no?

**Review by brifsttar — CHANGES_REQUESTED** [review]
Quelques retours, mais grand merci pour toutes ces corrections !

File: _posts/2022-09-05-workflow-2.md (hunk 1 above)
Comment (rderollepot) [reply _posts/2022-09-05-workflow-2.md]: Oh yeah, I think I missed the _'re_

File: _posts/2023-05-01-whats-new-2023-05.md (hunk 3 above)
Comment (rderollepot) [reply _posts/2023-05-01-whats-new-2023-05.md]: Yeah also, I must have suggested that while reviewing the typos PyCharm found, not having the full sentence in mind anymore...

File: _posts/2023-03-01-mirrors.md (hunk 2 above)
Comment (rderollepot) [reply _posts/2023-03-01-mirrors.md]: set up (verb) instead of setup (noun)

Comment (brifsttar) [reply _posts/2023-03-01-mirrors.md]: But it's in an embedded tweet though, I think it might be better to leave the typo since it's in a the Tweet anyway.

Comment (rderollepot) [reply _posts/2023-03-01-mirrors.md]: Indeed, I did not realize it was in a tweet

**Comment by brifsttar** [comment]
Merci beaucoup !

**Pull request merged**